Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -994,36 +1001,118 @@
         page->chrome().print(*m_frame->coreLocalFrame());
 }
 
-#if PLATFORM(MAC)
+std::optional<PageIdentifier> PDFPluginBase::pageIdentifier() const
+{
+    return m_frame && m_frame->coreLocalFrame() ? m_frame->coreLocalFrame()->pageID() : std::nullopt;
+}
 
-void PDFPluginBase::writeItemsToPasteboard(NSString *pasteboardName, Vector<PasteboardItem>&& pasteboardItems) const
+NSString *PDFPluginBase::stringPasteboardType()
 {
-    // FIXME: <https://webkit.org/b/269174> PDFPluginBase::writeItemsToPasteboard should be platform-agnostic.
-    auto pasteboardTypes = pasteboardItems.map([](const auto& item) -> String {
-        return item.type.get();
-    });
-    auto pageIdentifier = m_frame && m_frame->coreLocalFrame() ? m_frame->coreLocalFrame()->pageID() : std::nullopt;
+#if PLATFORM(IOS_FAMILY)
+    return UTTypeText.identifier;
```

Comment:
Nit - we generally use `UTTypeUTF8PlainText` instead of `UTTypeText`, since the latter is an abstract type that encompasses HTML and other rich text formats too.

(That said, I think `PlatformPasteboardIOS` ends up correcting this anyways by mapping `Text` to `UTF8PlainText` so this also works in practice, albeit with a little indirection)